### DISCUSSION TITLE
how to creat personal templates?

### DISCUSSION THREAD

**Discussion by sagrrecn** [dani-garcia/vaultwarden#4395] [General]
i have  set Reload templates  true ,how to creat personal templates?

**Comment by stefan0xC** [comment]
What do you mean by _personal_ templates? You can adapt the [handlebars](https://docs.rs/handlebars/latest/handlebars/) files in the [`src/static/templates` folder](https://github.com/dani-garcia/vaultwarden/tree/main/src/static/templates) by providing your own but that's more for customizing the mails, the 404 page and `/admin` pages. Cf. the [translating the email templates wiki page](https://github.com/dani-garcia/vaultwarden/wiki/Translating-the-email-templates) for a small tutorial.
